tests/python-pytest/onnx/export/onnx_backend_test.py

```diff
@@ -74,15 +74,14 @@
     'test_globalmaxpool',
     'test_globalaveragepool',
     # enabling partial test cases for matmul
```

Comment:
Can you remove this comment if all matmul test cases are enabled now?

Reply:
done